modules/premium-blocks.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Block Editor functionality for Premium Blocks.
+ *
+ * @package Jetpack
+ */
+
+class Premium_Blocks {
```

Comment:
I would recommend either prefixing that class name or putting it in the `\Automattic\Jetpack\Extensions\` namespace, to avoid any conflicts with third-party plugins.